blazor/rating/selection.md

```diff
@@ -45,9 +42,7 @@ Rated Value : @value
 
 ## Minimum value
 
-You can use the [Min](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_Min) property to set the lowest possible rating.
-
-Below example demonstrates the Min Value of Rating.
+The Blazor Rating component provides support for setting the minimum possible rating value using the [Min](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfRating.html#Syncfusion_Blazor_Inputs_SfRating_Min) property. You can use this property to specify the lowest rating value that the user can select. If you set the `Min` property to 2, then you will not be able to select a rating lower than 2.
```

Comment:
Rephrase, First 2 sentences appears like 2 explanation

Reply:
updated the content as per comments.